debug/src/debug.js

```diff
@@ -3,6 +3,14 @@ import { getDisplayName } from './devtools/custom';
 import { options } from 'preact';
 import { ELEMENT_NODE, DOCUMENT_NODE, DOCUMENT_FRAGMENT_NODE } from './constants';
 
+function getClosestDomNodeParent(parent) {
+	if (!parent) return null;
+	if (typeof parent.type === 'function') {
+		return getClosestDomNodeParent(parent._parent);
+	}
+	return parent.type;
```

Comment:
Minor nit: I'd prefer it to just return the parent `vnode` instead of the `type` only. It feels more in line with the name of the function and may allow it to be used elsewhere :+1: